docs/reference/8-admin-app/whats-new.md

```diff
@@ -0,0 +1,290 @@
+# What's New
+
+This section provides an overview of what's new in the latest versions of the ODS / API Admin App for Technical Suite Two and Technical Suite Three.
+
+**Contents**
+
+* [Updates in Admin App v3.3](#updates-in-admin-app-v33)
+* [Updates in Admin App v3.2](#updates-in-admin-app-v32)
+* [Updates in Admin App v3.1](#updates-in-admin-app-v31)
+* [Updates in Admin App for Suite 3 v3.0](#updates-in-admin-app-for-suite-3-v30)
+* [Updates in Admin App for Suite 3 v2.4](#updates-in-admin-app-for-suite-3-v24)
+* [Updates in Admin App for Suite 3 v2.3](#updates-in-admin-app-for-suite-3-v23)
+* [Updates in Admin App for Suite 3 v2.2.1](#updates-in-admin-app-for-suite-3-v221)
+* [Updates in Admin App for Suite 3 v2.2](#updates-in-admin-app-for-suite-3-v22)
+* [Updates in Admin App for Suite 3 v2.1.0](#updates-in-admin-app-for-suite-3-v210)
+* [Updates in Admin App for Suite 3 v2.0.1](#updates-in-admin-app-for-suite-3-v201)
+* [Updates in Admin App for Suite 3 v2.0.0](#updates-in-admin-app-for-suite-3-v200)
+* [Updates in Admin App v1.8 for Ed-Fi ODS / API v3.4](#updates-in-admin-app-v18-for-ed-fi-ods-api-v34)
+* [Updates in Admin App v1.7 for Ed-Fi ODS / API v3.3](#updates-in-admin-app-v17-for-ed-fi-ods-api-v33)
+* [Updates in Admin App v1.6 for Ed-Fi ODS / API v3.2](#updates-in-admin-app-v16-for-ed-fi-ods-api-v32)
+* [Updates in Admin App v1.5.1 for Ed-Fi ODS / API v2.6](#updates-in-admin-app-v151-for-ed-fi-ods-api-v26)
+* [Updates in Admin App v1.5](#updates-in-admin-app-v15)
```

Comment:
I've typically been deleting the "Contents" section that came over from Confluence, because we now have the improved table of contents auto-generated on the right side of the page.